test_naucse_render/fixtures/expected-dumps/courses/serial-test.yaml

```diff
@@ -0,0 +1,51 @@
+api_version:
+- 0
+- 1
+course:
+    sessions:
+    -   serial: '1'
+        slug: autonumber-1
+        source_file: courses/serial-test/info.yml
+        title: Autonumber 1
+    -   serial: '2'
+        slug: autonumber-2
+        source_file: courses/serial-test/info.yml
+        title: Autonumber 2
+    -   serial: None
```

Comment:
not sure if this is supposed to be None, null or !!null.

Reply:
Gah! YAML syntax got me here :(

Reply:
Thanks for catching this!